diff --git a/bubblewrap.c b/bubblewrap.c
index 8322ea0..4e20262 100644
--- a/bubblewrap.c
+++ b/bubblewrap.c
@@ -868,13 +868,6 @@ acquire_privs (void)
   /* Keep only the required capabilities for setup */
   set_required_caps ();
 }
-  else if (real_uid != 0 && has_caps ())
-  {
-    /* We have some capabilities in the non-setuid case, which should not happen.
-     * Let's drop them. */
-    drop_all_caps (FALSE);
-    is_privileged = FALSE;
-  }

   /* TODO: Handle the case where we have no permissions */
 }
